adapters/restclient/lib/opentelemetry/adapters/restclient/adapter.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module RestClient
+      class Adapter < OpenTelemetry::Instrumentation::Adapter
+        install do |config|
+          require_dependencies
```

Comment:
I'm taking a mental note of how. many times I see this in adapters. It might make sense to make this an optional part of the DSL at some point.